index.html

```diff
@@ -4,9 +4,45 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <title>Day 1: Solo Build</title>
+    <link rel="stylesheet" href="./src/styles.css">
 </head>
 <body>
-    
+    <H1>Hi I'm Eli, I like <a href="#pets">my pets</a> and <a href="#kayak">kayaking!</a> Below are my pets and a couple fun  words about kayaking.</H1>
+    <h2 id="pets">My Pets are and look like:</h2>
```

Comment:
It's a good idea semantically to wrap your sections in a section or article tag depending on the content.